docker/config/l2-node-besu/l2-node-besu-config.toml

```diff
@@ -63,3 +63,5 @@ bonsai-historical-block-limit=1024
 
 Xin-process-rpc-enabled=true
 Xin-process-rpc-apis=["ETH", "MINER"]
+
+Xbonsai-parallel-tx-processing-enabled=false
```

Comment:
@fab-10 so this mean we'd also need to disable "parallel-tx-processing" when we deploy this `linea-besu-package` image on our production environments, until we have a new release with the fix, right?